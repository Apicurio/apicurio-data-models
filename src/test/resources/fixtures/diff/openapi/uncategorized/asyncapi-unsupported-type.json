{
  "asyncapi": "2.0.0",
  "info": {
    "title": "Account Service",
    "version": "1.0.0",
    "description": "Manages user accounts in the system.\n",
    "license": {
      "name": "Apache 2.0",
      "url": "https://www.apache.org/licenses/LICENSE-2.0"
    }
  }
}