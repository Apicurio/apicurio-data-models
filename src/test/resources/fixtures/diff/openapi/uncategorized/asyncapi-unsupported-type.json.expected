EXCEPTION => class java.lang.RuntimeException :: Diff Visitor does not support the document type 'asyncapi2'